pkg/sql/drop_index.go

```diff
@@ -245,6 +245,14 @@ func (p *planner) dropIndexByName(
 		return nil
 	}
 
+	if idx.Unique && behavior != tree.DropCascade && constraintBehavior != ignoreIdxConstraint && !idx.CreatedExplicitly {
+		return errors.WithHint(
+			pgerror.Newf(pgcode.DependentObjectsStillExist,
```

Comment:
optional nit: prefer arguments one line at a time (neater in most cases + nicer on git blame)

```
pgerror.Newf(
  pgcode.DependentObjectsStillExist,
	"index %q is in use as unique constraint",
  idx.Name,
),
```